### ISSUE TITLE
mob status should show if a mob session exists

### ISSUE THREAD

**Issue by oobenland** [remotemobprogramming/mob#410]
It would be helpful when `mob status` shows if a mob session exists on the current branch, when not on a mob branch, to look up if a remote mob branch already exists.

Something like this:

    ➜  myProject git:(main) mob status
    > you are on base branch 'main'
    > wip branch 'mob/main' contains 7 commits. The last 5 were:
    > 38209edb 6 hours ago <Wanda Hess>
    > 2359e605 5 hours ago <Ross Carter>
    > 3868b5c6 5 hours ago <Claudia Osborne>
    > b848355a 5 hours ago <Lee Klein>
    > 209e8bb2 5 hours ago <Ammar Carlson>

**Comment by hollesse** [comment]
Hi @oobenland sounds like a good idea. Would you like to create a pull request?

**Comment by hollesse** [comment]
As I just talked to @oobenland we found out that the feature is already there and he was just in a project where no mob session was open. The output of the requested feature which is already implemented looks like this
```
> you are on base branch 'main'
> remote wip branches detected:
  - origin/mob/main
  - origin/mob/main-my-feature
```

For @oobenland it is important to know how old these mob branches are. Therefore we should check if we could get a date maybe even datetime of the last commit.
